client/src/main/scala/drt/client/components/FlightComponents.scala

```diff
@@ -59,10 +59,11 @@ object FlightComponents {
     val port: String = if (flight.ActPax > 0) flight.ActPax.toString else "n/a"
     val last: String = flight.LastKnownPax.getOrElse("n/a").toString
     val max: String = if (flight.MaxPax > 0) flight.MaxPax.toString else "n/a"
+    val PortTotalPax = (flight.ActPax + flight.TranPax)
```

Comment:
Please lowercase the first letter of variable names. Upper case by convention is just used for class and object names